### PR TITLE
fix: [CDS-77684]: Handle null point exception if created by field is nil

### PR DESCRIPTION
**Pull request by sarthakkasat**
## Describe your changes
Handle null point exception if created by field is nil

## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`

### CODE REVIEW

File: internal/service/platform/file_store/file_store_utils.go [hunk 1]
```diff
@@ -79,5 +79,16 @@ func getOptionalString(str interface{}) optional.String {
 	return optional.NewString(v)
 }
 
+func getSafeEmail(user *nextgen.EmbeddedUserDetailsDto) string {
```
Comment (yogesh-chauhan) [inline internal/service/platform/file_store/file_store_utils.go:82]: Lets just call it getEmail 
methods are supposed to be safe anyways :)

Comment (sarthakkasat) [reply internal/service/platform/file_store/file_store_utils.go]: Updated the name :)

**Review by yogesh-chauhan — DISMISSED** [review]

**Comment by sarthakkasat** [comment]
trigger subcategoryfieldcheck

**Comment by sarthakkasat** [comment]
trigger build

**Comment by yogesh-chauhan** [comment]
We also need to add changelog @sarthakkasat

**Review by yogesh-chauhan — APPROVED** [review]

**Comment by yogesh-chauhan** [comment]
trigger build

**Comment by yogesh-chauhan** [comment]
trigger subcategoryfieldcheck

**Comment by yogesh-chauhan** [comment]
trigger build

**Comment by yogesh-chauhan** [comment]
trigger build

**Comment by yogesh-chauhan** [comment]
trigger build

**Comment by yogesh-chauhan** [comment]
trigger build

**Comment by yogesh-chauhan** [comment]
trigger subcategoryfieldcheck

**Pull request merged**